### DISCUSSION TITLE
Mirsvit Google tag manager is not workin on PDP page

### DISCUSSION THREAD

**Discussion by arunguru77** [breezefront/community#197] [1. Help]
Hi,
we are using Mirasvit module for capture GA4. it not working on PDP page. we connect the with mirasvit they replied it issue of breeze theme.
we checked same on Luma its woking fine.
https://mirasvit.com/magento-2-google-tag-manager.html
PDP page is https://artuk.wcltest.com/john-lavery-under-the-cherry-tree-placemat.html
any help here

**Comment by dimdimich** [comment]
Hi,

do you have this integration module installed https://github.com/breezefront/module-breeze-mirasvit-gtm  ?
